inlong-manager/manager-client/src/main/java/org/apache/inlong/manager/client/api/impl/InlongGroupImpl.java

```diff
@@ -267,7 +269,7 @@ private InlongGroupContext generateSnapshot() {
                 logList.stream().filter(x -> StringUtils.isNotEmpty(x.getComponentName()))
                         .forEach(streamLog -> {
                             String componentName = streamLog.getComponentName();
-                            String log = JsonUtils.toJsonString(streamLog);
+                            String log = toJsonString(streamLog);
```

Comment:
No need to use a static import, just use `JsonUtils.toJsonString` is more clearly.